weed/pb/grpc_client_server.go

```
@@ -401,6 +401,13 @@ func shouldInvalidateConnection(ctx context.Context, err error) bool {
 		return false
 	}
 
+	// gRPC raises this locally, before the RPC reaches the wire, when this
+	// process already closed the ClientConn. The caller is a bystander of
+	// someone else's teardown, and knows nothing about the peer.
+	if errors.Is(err, grpc.ErrClientConnClosing) {
```

golangci-lint (2.12.2)

[error] SA1019: grpc.ErrClientConnClosing is deprecated: this error should not be relied upon by users; use the status code of Canceled instead. 

(staticcheck)